Add new file for lion details

sabatjohn17 opened the pull request:
Created a new text file to include information about lions.

sabatjohn17 committed:
Add new file for lion details

Created a new text file to include information about lions.

ajit-spec reviewed (changes requested):
add few lines to lion.txt file

sabatjohn17 committed:
Add info about lions and their habitat

- Added a brief description of lions
- Mentioned their status as "king of the jungle"
- Highlighted their natural habitat in savannas

ajit-spec reviewed (approved):
good
